lib/widgets/game_view.dart

```diff
@@ -84,6 +101,42 @@ class _GameViewState extends State<GameView> {
           },
           child: const Text('Choose grid'),
         ),
+        const SizedBox(height: 20),
+        Visibility(
+            visible: winner != '',
```

Comment:
We are not detecting winner in this PR. Please remove the code related to it.

Reply:
removed